tests/ginkgo_reporter.go

```diff
@@ -0,0 +1,24 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/reporters"
+	"github.com/onsi/ginkgo/v2/types"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+func GenerateGinkgoJunitReport(reportName string, report types.Report) {
+	logger := zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter))
```

Comment:
I guess the `DevMode` should be removed before merging

Reply:
This is only used for the generation of the junit reports, we can remove it, but we have it in istio as well and I don't see any issue with having more extensive logging in this case.
[Docs](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/log/zap@v0.16.2#Options.Development)